data/categories.txt

```diff
@@ -22,6 +22,7 @@ category "ship"
 	"Light Warship"
 	"Medium Warship"
 	"Heavy Warship"
+	"Superheavy"
```

Comment:
This confuses me. I haven't looked into this code-wise, but why is the new category added near the bottom here but appearing at the top of the shipyard? This seems to be one of two things:

1. This got changed between when the screenshot was taken and when the commit was made.
1. The way the code works this will still display at the top, in which case it should probably still be moved to the top here.

Reply:
I believe it's something the code does, because #1 here is not the case. If it helps for clarification I can just move it up in the lines yes.

Reply:
That would definitely help clarify things, yes, since all the other categories are in the order they appear in the shipyard in.

Reply:
The category appears at the top in your game despite being defined at the button because of your plugins. In vanilla, this would appear at the bottom.

Reply:
Oh, well that makes sense then. Thoughts on where this should be? Below Heavy Warships? At the very bottom? Near Utility?

Reply:
I think below heavy warship is fine.